app/root.tsx

```diff
@@ -1,3 +1,4 @@
+import { ThemeProvider } from "@emotion/react";
```

Comment:
This should probably be from "@mui/material" and not from Emotion.

Reply:
Good eye! Fixed! 👍